src/components/Header.tsx

```
@@ -1,20 +1,77 @@
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from "@/components/ui/dropdown-menu"
 
 export function Header() {
+  const [nomeUsuario, setNomeUsuario] = useState<string | null>(null)
+
+  useEffect(() => {
+    const usuarioStr = localStorage.getItem("usuario")
```

ESLint

[error] Replace `"usuario"` with `'usuario'`

(prettier/prettier)